/**:
  ros__parameters:
    CameraExtrinsicMat: [0.0155420096457260,	-0.998979157569557,	-0.0424156654762087,	-0.0140384849280508,
0.1730289788462245,	0.0434416230352977,	-0.996383256401239,	0.717764190124461,
0.997208711446531,	0.0123882254450174,	0.0736295978902069,	0.,
0.,	0.,	0.,	1.]
    CameraMat: [465.794342133086,	0.,	327.279450927008,
0.,	466.058095933375,	249.152907513456,
0.,	0.,	1.]
    DistCoeff: [-0.370974152435954,	0.143886871117310, 0.,   0.]
    #0913 flir
